### PR TITLE
a few more

### PR DESCRIPTION
**Pull request by jeffwilde**
more test

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jeffwilde/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: SomeNewData.txt [hunk 1]
```diff
@@ -1 +1,13 @@
 test
+
+1
+
+2
+
+3
```
Comment (jeffwilde) [inline SomeNewData.txt]: !fix needed here

File: SomeNewData.txt [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+4
+
```
Comment (jeffwilde) [inline SomeNewData.txt:10]: !resolve needed here

Comment (jeffwilde) [inline SomeNewData.txt:10]: !resolved

File: SomeNewData.txt (hunk 1 above)
Comment (jeffwilde) [inline SomeNewData.txt]: !fixed

File: SomeNewData.txt [hunk 3]
```diff
@@ -1 +1,13 @@
 test
+
+1
+
+2
+
+33
```
Comment (jeffwilde) [inline SomeNewData.txt:7]: !fix  !resolve

File: SomeNewData0.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+test
+
+1
+
+2
+
+33
+
+4
```
Comment (jeffwilde) [inline SomeNewData0.txt]: !resolve

File: SomeNewData.txt (hunk 3 above)
Comment (jeffwilde) [inline SomeNewData.txt:7]: !fixed  !resolved

File: SomeNewData0.txt (hunk 4 above)
Comment (jeffwilde) [inline SomeNewData0.txt]: !resolved

**Pull request merged**